### ISSUE TITLE
Add "Open Lock" Icon

### ISSUE THREAD

**Issue by mistryrn** [icgc-argo/uikit#26]
## Detailed Description
Platform UI's File Entity page requires an "Open Lock" icon for files with "Open" access.
https://github.com/icgc-argo/platform-ui/issues/2090

To prevent forking changes between repos, the icon has not been added to the previous UIKit. Once this repo is ready, the icon will be added and a follow-up ticket in platform-ui will be created to use the new icon there.

Icon file is attached in the zip below:
[icon-unlock.svg.zip](https://github.com/icgc-argo/uikit/files/7671772/icon-unlock.svg.zip)
